src/Codeception/Coverage/PhpCodeCoverageFactory.php

```diff
@@ -4,25 +4,23 @@
 
 namespace Codeception\Coverage;
 
+use Codeception\PHPUnit\Compatibility\PHPUnit10;
 use PHPUnit\Runner\CodeCoverage as PHPUnitCodeCoverage;
 use SebastianBergmann\CodeCoverage\CodeCoverage;
 use SebastianBergmann\CodeCoverage\Driver\Driver;
 use SebastianBergmann\CodeCoverage\Filter as CodeCoverageFilter;
-use function method_exists;
 
 class PhpCodeCoverageFactory
 {
     public static function build(): CodeCoverage
     {
         $filter = new CodeCoverageFilter();
-        if (method_exists(PHPUnitCodeCoverage::class, 'activate')) {
-            // php-code-coverage 10
+        if (PHPUnit10::codeCoverageRunnerClassExists()) {
```

Comment:
I changed this because apparently that class only exists in `PHPUnit 10`, first you were using the `isActive` method which could also be non-existent... So I think validating this makes more sense, am I correct?